design/baremetal-operator/implicit-boot-mode.md

```diff
@@ -141,7 +141,7 @@ values later for other hardware platforms.
 ## Design Details
 
 - Update the [AccessDetails
-  interface](https://github.com/metal3-io/baremetal-operator/blob/main/pkg/bmc/access.go#L27)
+  interface](https://github.com/metal3-io/baremetal-operator/blob/dabe5e14bafa00db6ccb37f1169c74ee3dac4425/pkg/hardwareutils/bmc/access.go)
```

Comment:
Please keep the line reference, update to point to correct line as needed.

Reply:
Ok, would fix that now

Reply:
It is now pointing to wrong line, correct line is L47.